pkgs/applications/version-management/gitstatus/default.nix

```diff
@@ -83,9 +102,26 @@ stdenv.mkDerivation rec {
 
   meta = with lib; {
     description = "10x faster implementation of `git status` command";
+    longDescription = ''
+      To enable a prompt using gitstatus directly, add one of the following to your NixOS configuration:
+      `programs.bash.promptInit = "source $(gitstatus-share)/gitstatus.prompt.sh";`
+      `programs.zsh.promptInit = "source $(gitstatus-share)/gitstatus.prompt.zsh";`
+
+      To enable from Home Manager:
+      `programs.bash.initExtra = "source $(gitstatus-share)/gitstatus.prompt.sh";`
+      `programs.zsh.initExtra = "source $(gitstatus-share)/gitstatus.prompt.zsh";`
+
```

Comment:
```suggestion
```
We won't notice in nixpkgs when this changes in home-manager.

Reply:
Removed